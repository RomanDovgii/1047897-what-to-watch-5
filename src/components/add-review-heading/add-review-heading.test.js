import React from "react";
import renderer from "react-test-renderer";
import AddReviewHeading from "./add-review-heading";

it(
    `Should render correct AddReviewHeading`,
    () => {
      const tree = renderer
      .create(
          <AddReviewHeading/>
      )
      .toJSON();

      expect(tree).toMatchSnapshot();
    }
);
