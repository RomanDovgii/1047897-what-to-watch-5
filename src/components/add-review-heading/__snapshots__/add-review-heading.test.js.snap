// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correct AddReviewHeading 1`] = `
<h1
  className="visually-hidden"
>
  WTW
</h1>
`;
